lib/internal/blocklist.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+const fs = require('fs');
```

Comment:
Better to make this lazy

```suggestion
let fs;
function lazyFs() {
  fs ??= require('fs');
  return fs;
}
```

The update the places where you're calling `fs` below, e.g. `lazyFs().existsSync(...)`